Objects/dictobject.c

```diff
@@ -2804,18 +2919,22 @@ dict_clear(PyDictObject *mp)
 }
 
 static PyObject *
-dict_pop(PyDictObject *mp, PyObject *args)
+dict_pop(PyDictObject *mp, PyObject *args, PyObject *kwargs)
```

Comment:
OK, I'll stop sharing methods and override it only for keyword argument support of odict.